integration-tests/ci/profiles.yml

```diff
@@ -7,18 +7,18 @@ integration_tests:
       project: "{{ env_var('GOOGLE_CLOUD_PROJECT') }}"
       dataset: dbt_unit_testing
       threads: 1
-    
+
     postgres:
       type: postgres
-      host: localhost
+      host: postgres
```

Comment:
Could you please revert this to localhost? It's needed for the CI env.

Reply:
Just now I saw this message. Already pushed the required change.